From d37a8b13fc6a97cd0a8f28c44c5843f0f032c087 Mon Sep 17 00:00:00 2001
From: "Song.Li" <Song.Li@windriver.com>
Date: Thu, 29 Nov 2012 15:15:13 +0800
Subject: [PATCH] remove hard coded include and lib paths

There are hardcode paths in configure.in
eg:/usr/include,$HOME/include and so on.
These paths will cause Makefile to search
host path when doing cross-compiling.
and bitbake will prompt this as an error.
so remove them.

Upstream-Status: Inappropriate [not a real bug,just for error prompt of yocto]

Signed-off-by: Song.Li <Song.Li@windriver.com>
---
 configure.in |   10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/configure.in b/configure.in
index bcfd5bc..5b24947 100644
--- a/configure.in
+++ b/configure.in
@@ -32,21 +32,15 @@ MYPCFILES="tokyocabinet.pc"
 
 # Building flags
 MYCFLAGS="-std=c99 -Wall -fPIC -fsigned-char -O2"
-MYCPPFLAGS="-I. -I\$(INCLUDEDIR) -I$HOME/include -I/usr/local/include"
+MYCPPFLAGS="-I."
 MYCPPFLAGS="$MYCPPFLAGS -DNDEBUG -D_GNU_SOURCE=1 -D_REENTRANT -D__EXTENSIONS__"
-MYLDFLAGS="-L. -L\$(LIBDIR) -L$HOME/lib -L/usr/local/lib"
+MYLDFLAGS="-L."
 MYCMDLDFLAGS=""
 MYRUNPATH="\$(LIBDIR)"
 MYLDLIBPATHENV="LD_LIBRARY_PATH"
 MYPOSTCMD="true"
 
 # Building paths
-PATH="$PATH:$HOME/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin"
-PATH="$PATH:/opt/SUNWspro/bin:/usr/ccs/bin:/usr/xpg4/bin:/usr/xpg6/bin:/usr/ucb"
-CPATH="$HOME/include:/usr/local/include:$CPATH"
-LIBRARY_PATH="$HOME/lib:/usr/local/lib:$LIBRARY_PATH"
-LD_LIBRARY_PATH="$HOME/lib:/usr/local/lib:$LD_LIBRARY_PATH"
-PKG_CONFIG_PATH="$HOME/lib/pkgconfig:/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH"
 export PATH CPATH LIBRARY_PATH LD_LIBRARY_PATH PKG_CONFIG_PATH
 
 
-- 
1.7.9.5

